app/app.css

```diff
@@ -23,3 +23,23 @@
   content: "|";
   padding-right: 0.3em;
 }
+
+
+
+.table > thead > tr> th{text-align: center;}
+.table > tbody > tr> td{text-align: center;}
+.btn.btn-default.btn-block{text-align: center;}
+
+.membersbound {background: #fff; color: #373737;}
+.membersbound >  div {display: none; padding: 20px 25px 5px;}
```

Comment:
Create a single class that uses attribute then apply it to any tags that need it. This is okay for things like tables, just to save some code. But, usually this is never used. Layered CSS gets really confusing and is usually not good practice.